libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/widgets/WPBottomSheetDialogFragment.java

```diff
@@ -0,0 +1,44 @@
+package org.wordpress.android.login.widgets;
+
+import android.app.Dialog;
+import android.os.Bundle;
+import android.view.WindowManager;
+
+import androidx.annotation.NonNull;
+
+import com.google.android.material.bottomsheet.BottomSheetDialog;
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment;
+
+import org.wordpress.android.login.R;
+
+public class WPBottomSheetDialogFragment extends BottomSheetDialogFragment {
+    @Override
+    public int getTheme() {
+        return R.style.LoginTheme_BottomSheetDialogStyle;
+    }
+
+    @NonNull
+    @Override
+    public Dialog onCreateDialog(Bundle savedInstanceState) {
+        setRetainInstance(true);
```

Comment:
Invoking `setRetainInstance(true)` on a fragment with UI is quite risky (we might/will leak all the views), do we really need it? Moreover, `SignupBottomSheetDialogFragment` currently holds a reference to the Activity, which will also leak after a configuration change. Wdyt?

Reply:
Fixed here 1d51b1f